From 770cefb11cf0aef72fc168379912492e51edd504 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Mon, 12 Mar 2018 18:57:48 +0800
Subject: [PATCH 4279/5242] MLK-17829-2 ARM64: imx8mscale: add s3508 touch
 support in device tree

commit  38882c2482b20fae3607e1658b3392e011a13a26 from
https://source.codeaurora.org/external/imx/linux-imx.git

Default disable this touch.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 2676b8e147cdf58e70fdb2a19253888879730b0f)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts  |   20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
index 91a7d38..78fa6d4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
@@ -114,5 +114,25 @@
 				MX8MQ_IOMUXC_ECSPI1_SCLK_GPIO5_IO6	0x16
 			>;
 		};
+
+		pinctrl_i2c1_synaptics_dsx_io: synaptics_dsx_iogrp {
+			fsl,pins = <
+				MX8MQ_IOMUXC_ECSPI1_MOSI_GPIO5_IO7              0x19
+			>;
+		};
+
+	};
+};
+
+&i2c1 {
+	synaptics_dsx_ts@20 {
+		compatible = "synaptics_dsx";
+		reg = <0x20>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_i2c1_synaptics_dsx_io>;
+		interrupt-parent = <&gpio5>;
+		interrupts = <7 8>;
+	/*	rest-gpios = <&gpio5 6 GPIO_ACTIVE_HIGH>;  */
+		status = "disabled";
 	};
 };
-- 
1.7.9.5

